src/content/developers/docs/data-availability.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Data availability 
+description: An overview of problems and solutions relating to data availability in Ethereum 
+lang: en
+sidebar: true
+---
+
+In Ethereum, a node proposing a new block is required to make the underlying data available to the remaining peer-to-peer nodes on the network. This concept, known as data availability, is fundamental to the security and functionality of every blockchain, including Ethereum. Data availability is so important that an entire field of research studying the so-called “data availability problem” exists. 
+
+## What is data availability? {#what-is-data-availability}
+
+“Data availability” is a guarantee that all transaction data for a block was published by the block proposer and is available to other network participants. As explained in “[Intro to Ethereum](/developers/docs/intro-to-ethereum/)”, Ethereum transactions are processed in [blocks](/developers/docs/blocks/) and chained together to form the “blockchain”.
+
+Each block has two major parts: 
+
+- The **block header**: This contains general information (metadata) about the block, such as the timestamp, block hash, block number, etc.
+- The  **block body**: This contains the actual transactions processed as part of the block. 
+
+[Miner nodes](/developers/docs/consensus-mechanisms/pow/mining/) (or validators in [proof-of-stake](/developers/docs/consensus-mechanisms/pos/)) proposing new blocks must publish both the block header and block body. This allows nodes participating in [consensus](/developers/docs/consensus-mechanisms/) (full nodes) to download the block and confirm that transactions follow Ethereum’s consensus protocol. 
+
+[Full nodes](/developers/docs/nodes-and-clients/#full-node) are different from [light nodes](/developers/docs/nodes-and-clients/#light-node) that only download the block header. Without full nodes verifying transactions, block proposers could get away with inserting malicious transactions in blocks. 
+
+### The data availability problem {#the-data-availability-problem}
+
+The data availability problem is encapsulated in one question: “How do we verify that the data for a newly produced block is actually available?” As mentioned, the security Ethereum rests on the assumption that full nodes have access to block data. 
+
+If a block producer publishes a new block, but hides some (or all) of the underlying information, they might be able to finalize invalid transactions. Even if a block is valid, withholding data still has negative implications for users and the functionality of the network.   
+
+Most early discussions around data availability concerned the visibility of transaction data to full nodes (validators) participating in Ethereum’s consensus. However, newer DA research is also focused on verifying data availability with light clients. For light clients, the problem of data availability concerns validating a block’s data availability without having to download the entire block. 
```

Comment:
I think an equally critical aspect is DA for zk and optimistic rollups

Reply:
and also how data will be provably available on a stateless Ethereum (e.g. Portal network development)

Reply:
Added new text touching on both points. LMK what you think.